File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldLogNumber.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdRole;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldLogNumber extends DefnFieldNumber
{
  @Nullable
  public Boolean hideInfo;

  @Nullable
  public MetaIdRole[] logReadRoleSet;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldLogNumber.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdRole

interface DefnFieldLogNumber : DefnFieldNumber {
    val hideInfo: Boolean?
    val logReadRoleSet: List<MetaIdRole>?
}
